helpers.md

```diff
@@ -153,20 +152,6 @@ The `array_except` method removes the given key / value pairs from the array:
 
 	// ['name' => 'Desk']
 
-<a name="method-array-fetch"></a>
-#### `array_fetch()` {#collection-method}
-
-The `array_fetch` method returns a flattened array containing the values of the specified nested element:
```

Comment:
Shouldn't we say, "This been replaced with `array_pluck`" or something... Since they might be people that been using it and now don't know what to do.